### PR TITLE
Fixes #71: Add set_stream_0 command for Techwin camera

### LINKED ISSUE #71: set_stream_0 for SNC-4241BE
maker: `Samsung Techwin`
command_id: `set_stream_0` 

2048kbps, 1080p, Baseline profile and 20 fps:

method: `GET`
auth_type: `basic`
regex_success: `.*result.*OK.*`
(which is sloppy match for the following json response)
```json
{
	"result":	"OK"
}
```

url:
```
http://{{username}}:{{password}}@{{ip_address}}/action/setConfig?name=MainStream&MainStream.VideoType=0&MainStream.BitrateType=1&MainStream.BitrateMode=1&MainStream.Fps=20&MainStream.CustomBitrate=2048&MainStream.Resolution=5&MainStream.VideoQuality=5&MainStream.LoadBalancing=0&MainStream.IFrameInterval=20&MainStream.AudioSwitch=0&MainStream.ProfileLevel=0
```

regex_firmware: `.*`
regex_model: `SNC-4241.*`

no `payload` since all parameters supplied in `url` above.

### PR DESCRIPTION
**Pull request by jhallard**

### CODE REVIEW

File: configs/configs.csv [hunk 1]
```diff
@@ -7,3 +7,5 @@ Hikvision,set_stream_01,http://{{username}}:{{password}}@{{ip_address}}/ISAPI/St
 Hikvision,set_stream_02,http://{{username}}:{{password}}@{{ip_address}}/ISAPI/Streaming/channels/102,PUT,basic,.*statusCode.*1.*,.*,.*DS-2CD2032-I.*,hikvision_ds-2cd2032-I_stream_02.xml
 Hikvision,enable_dhcp,http://{{username}}:{{password}}@{{ip_address}}/ISAPI/System/Network/interfaces/1,PUT,basic,.*statusCode.*1.*,.*,DS-2CD2032-I,hikvision_ds-2cd2032-I_enable_dhcp.xml
 Hikvision,reboot,http://{{username}}:{{password}}@{{ip_address}}/ISAPI/System/reboot,PUT,basic,.*statusCode.*1.*,.*,.*DS-2CD2032-I.*,
+Samsung Techwin,set_stream_0,http://{{username}}:{{password}}@{{ip_address}}/action/setConfig?name=MainStream&MainStream.VideoType=0&MainStream.BitrateType=1&MainStream.BitrateMode=1&MainStream.Fps=20&MainStream.CustomBitrate=2048&MainStream.Resolution=5&MainStream.VideoQuality=5&MainStream.LoadBalancing=0&MainStream.IFrameInterval=20&MainStream.AudioSwitch=0&MainStream.ProfileLevel=0,GET,basic,.*result.*OK.*,*,SNC-4241.*,
```
Comment (jcmaslan) [inline configs/configs.csv]: regex_firmware should be `.*` (not `*`)
Also remove empty line 11

**Review by jcmaslan — CHANGES_REQUESTED** [review]

File: configs/configs.csv (hunk 1 above)
Comment (jhallard) [reply configs/configs.csv]: fixed with 74493a4

**Comment by jhallard** [comment]
@jcmaslan this now includes the Techwin command and the Duhua camera.

**Comment by jcmaslan** [comment]
LGTM!

**Review by jcmaslan — APPROVED** [review]

**Pull request merged**